driver/driver_test.go

```diff
@@ -1103,3 +1104,34 @@ func TestInsertByteArray(t *testing.T) {
 		t.Fatalf("Expected JSON to be '%s', got '%s'", string(j), s)
 	}
 }
+
+func TestInsertByteArrayODBC(t *testing.T) {
+	log.UseTestLogger(t)
+
+	db, err := sql.Open("ramsql", "TestInsertByteArrayODBC")
+	if err != nil {
+		t.Fatalf("sql.Open : Error : %s\n", err)
+	}
+
+	_, err = db.Exec(`CREATE TABLE test_json (sequence_number BIGSERIAL PRIMARY KEY, json JSON, created_at TIMESTAMP)`)
+	if err != nil {
+		t.Fatalf("sql.Exec: Error: %s\n", err)
+	}
+
+	j, _ := json.Marshal(map[string]string{"a": "a"})
+
+	_, err = db.Exec("INSERT INTO test (json) values (?)", j)
```

Comment:
table name should be "test_json" here, that's why tests are failing

Reply:
Sorry about that. I fixed it.